types/express-as-promise/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "WebWorker"
```

Comment:
This is why the bot says “check config.” What are you using from webworker and could it be DOM instead? This is the second time I’ve seen this this week 🤔 Trying to figure out if there’s something we need to change in the infrastructure.

Reply:
You're right. I changed that to DOM